### PR TITLE
docs: fix phpstan ignored errors in app/ and admin/starter/tests/

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
- add checking `admin/starter/tests` by PHPStan
- add PHPDoc types

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: app/Config/Logger.php [hunk 1]
```diff
@@ -72,6 +72,8 @@ class Logger extends BaseConfig
      *
      * Handlers are executed in the order defined in this array, starting with
      * the handler on top and continuing down.
+     *
+     * @var array<string, array<string, int|list<string>|string>>
```
Comment (datamweb) [inline app/Config/Logger.php:76]: `FileHandler::class` is string or class-string?

File: app/Config/Routing.php [hunk 2]
```diff
@@ -106,7 +108,7 @@ class Routing extends BaseRouting
      *       'blog' => 'Acme\Blog\Controllers',
      *   ]
      *
-     * @var array [ uri_segment => namespace ]
+     * @var array<string, string> [ uri_segment => namespace ]
```
Comment (datamweb) [inline app/Config/Routing.php:111]: The example(E.g.) is clear to me, I think `[ uri_segment => namespace ] ` is not needed.

File: system/Config/Routing.php [hunk 3]
```diff
@@ -104,7 +106,7 @@ class Routing extends BaseConfig
      *       'blog' => 'Acme\Blog\Controllers',
      *   ]
      *
-     * @var array [ uri_segment => namespace ]
+     * @var array<string, string> [ uri_segment => namespace ]
```
Comment (datamweb) [inline system/Config/Routing.php:109]: Comment above.

File: app/Config/Routing.php (hunk 2 above)
Comment (kenjis) [reply app/Config/Routing.php]: Removed.

File: system/Config/Routing.php (hunk 3 above)
Comment (kenjis) [reply system/Config/Routing.php]: Removed.

**Review by datamweb — APPROVED** [review]
LGTM!

**Pull request merged**